assets/js/main.js

```diff
@@ -1,12 +1,14 @@
-/* global $, ga, cxApi, _BLF */
+/* global ga, cxApi, _BLF */
 'use strict';
 
+const $ = require('jquery');
+
 // initialise router (eg. run conditional code for certain URLs)
 const Grapnel = require('./libs/grapnel');
 const router = new Grapnel({ pushState: true });
 
 // initialise Vue
-const Vue = require('../../node_modules/vue/dist/vue.common');
+const Vue = require('vue');
```

Comment:
This is wrong – look in devtools and you'll see the warnings!

Reply:
Accidental hold-over from some webpack experimentation. Reinstated now.